scripts/README.rst

```diff
@@ -0,0 +1,131 @@
+Google Cloud Key Management Service Python Samples
```

Comment:
Do we have a Single Tenant HSM public doc page? If we do it would be cool to link it here, otherwise feel free to ignore for now